core/locale/application/date_time_service.go

```diff
@@ -8,22 +8,20 @@ import (
 	"github.com/pkg/errors"
 )
 
-type (
-	// DateTimeServiceInterface to define a service to obtain formatted date time data
-	DateTimeServiceInterface interface {
-		GetDateTimeFormatterFromIsoString(dateTimeString string) (*domain.DateTimeFormatter, error)
-		GetDateTimeFormatter(dateTime time.Time) (*domain.DateTimeFormatter, error)
-	}
+// DateTimeServiceInterface to define a service to obtain formatted date time data
+type DateTimeServiceInterface interface {
+	GetDateTimeFormatterFromIsoString(dateTimeString string) (*domain.DateTimeFormatter, error)
+	GetDateTimeFormatter(dateTime time.Time) (*domain.DateTimeFormatter, error)
+}
 
-	// DateTimeService is a basic support service for date/time parsing
-	DateTimeService struct {
-		dateFormat     string
-		timeFormat     string
-		dateTimeFormat string
-		location       string
-		logger         flamingo.Logger
-	}
-)
+// DateTimeService is a basic support service for date/time parsing
+type DateTimeService struct {
+	dateFormat     string
+	timeFormat     string
+	dateTimeFormat string
+	location       string
+	logger         flamingo.Logger
+}
```

Comment:
What is benefit of not grouping type definitions inside type () ?

Reply:
In general, if we want to use locale module to expose it with swagger spec, most of the libraries have an issue to fetch types defined in brackets pairs (like go-swagger). Second reason would be to have it consistent, as in most of the cases we use it without brackets. :)